src/components/ProjectCard.tsx

```diff
@@ -165,11 +169,18 @@ export default function ProjectCard({
               </span>
             </div>
 
-            {metadata?.description && (
+            {tags?.length ? (
```

Comment:
on the project card, to avoid making the content too tall, we'll show tags if they exist—otherwise we'll show description. but not both